src/components/TokenBottomSheet.tsx

```diff
@@ -36,9 +38,11 @@ interface Props<T extends TokenBalance> {
   tokens: T[]
   TokenOptionComponent?: React.ComponentType<TokenOptionProps>
   showPriceUsdUnavailableWarning?: boolean
+  filterChips?: FilterChip<TokenBalance>[]
+  preSelectedFilterChips?: FilterChip<TokenBalance>[]
```

Comment:
nit: this naming seems more familiar in a React component:
```suggestion
  initialSelectedChipIds?: string[]
```